apps/website/src/components/ProjectsList.tsx

```diff
@@ -12,32 +12,18 @@ import { chunkArray } from "../utils/chunkArray"
 import { getProjectCategories } from "../utils/getProjectCategories"
 
 export default function ProjectsList(props: any) {
-    const [projects, setProjects] = useState<(typeof allProjects)[]>(chunkArray(allProjects))
+    const [projects] = useState<(typeof allProjects)[]>(chunkArray(allProjects))
     const [index, setIndex] = useState<number>(0)
-    const [selectedCategory, setSelectedCategory] = useState<string | null>(null)
     const [onlyPSE, setOnlyPSE] = useState<boolean | null>(null)
+    const [selectedCategories, setSelectedCategories] = useState<Array<string>>([])
 
-    const filterProjects = useCallback(() => {
```

Comment:
I think it's better to keep this function here and make it work with multi-categories. There were two different filters: 1 for PSE projects and 1 for tags and they should work together (that's why the logic is a little bit more complex than 1 filter). With your function, you're only filtering by tags/categories (nothing happens if I click on PSE/Community).

Reply:
You right, fixed in a6fde2fdedb3ad70970e643b4a212364ea77c6fa